src/common/libflux/handle.h

```diff
@@ -77,11 +77,11 @@ enum {
  */
 flux_t *flux_open (const char *uri, int flags);
 
-/* Like flux_open(), but if optional flux_error_t parameter is non-NULL,
+/* Like flux_open(), but if optional 'error' parameter is non-NULL,
  * then any errors normally emitted to stderr will instead be returned
- * in errp->text.
+ * in error->text.
  */
-flux_t *flux_open_ex (const char *uri, int flags, flux_error_t *errp);
+flux_t *flux_open_ex (const char *uri, int flags, flux_error_t *error);
```

Comment:
looks like a rebase error that lead to this change?  minimally should not be in this commit.

Reply:
Yes, thanks for catching that! Fixed.